source/schema/load-schema-from-drdl.txt

```diff
@@ -35,7 +35,8 @@ following actions:
 When you're done editing your ``.drdl`` file, you can either:
 
 - Use :binary:`~bin.mongodrdl` to :commandoption:`upload` the schema
-  to your MongoDB deployment.
+  to your MongoDB deployment. This requires naming your schema file,
+  using the :commandoption:`Name <name-schema>` option.
```

Comment:
```suggestion
  using the :commandoption:`name-schema <name-schema>` command.
```

I think it makes it clearer that the flow is 
1. upload schema
2. name schema
3. Use schema with mongosqld

Reply:
Updated. Rewrote as:

```
When you're done editing your ``.drdl`` file:

1. Use :binary:`~bin.mongodrdl` to :commandoption:`upload` the schema
  to your MongoDB deployment.
  
2. Add a name to your schema file, using the :commandoption:`Name <name-schema>` option.

3. Run :binary:`~bin.mongosqld` with the
  :option:`--schema <mongosqld --schema>` option to specify the
  ``<my-schema-name>.drdl``.
  
  ```